components/brave_news/browser/channel_migrator.cc

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/brave_news/browser/channel_migrator.h"
+
+#include <string_view>
+
+#include "base/containers/fixed_flat_map.h"
+#include "base/strings/strcat.h"
+#include "brave/components/brave_news/common/pref_names.h"
+#include "components/prefs/scoped_user_pref_update.h"
+
+namespace brave_news {
+
+namespace {
+
+constexpr auto kMigrateChannels =
+    base::MakeFixedFlatMap<std::string_view, std::string_view>(
+        {{"Tech News", "Technology"},
+         {"Sport", "Sports"},
+         {"Entertainment", "Celebrity News"},
```

Comment:
@LorenzoMinto I've used `Celebrity News` here instead of `Celebrities` as we don't seem to have a `Celebrity News` channel. 

I can update this to `Celebrities` and everything should work but wanted to double check.

Reply:
Let's go with Celebrities

Reply:
Hmm, if possible, I'd prefer to just migrate to one channel - not sure why but it scares me a little doing the split :D

Reply:
I can make the multi channel migration work, but it's **marginally** easier not to, if that's okay?

Reply:
I guess it would be fine to have `Entertainment News -> Entertainment` if that makes it easier. Only issue I see with that is that people that subscribed to Entertainment News for the gossip/celebs content will have to subscribe to the new channel, but that's not a big deal imo

Reply:
Thanks @LorenzoMinto - it does simplify things on my end a bit